src/lib/features/feature-search/feature.search.e2e.test.ts

```diff
@@ -240,8 +270,14 @@ test('should return all feature tags', async () => {
             {
                 name: 'my_feature_a',
                 tags: [
-                    { type: 'simple', value: 'my_tag' },
-                    { type: 'simple', value: 'second_tag' },
+                    {
+                        type: 'simple',
+                        value: 'my_tag',
+                    },
+                    {
+                        type: 'simple',
+                        value: 'second_tag',
+                    },
```

Comment:
Nothing related to this PR, but I'm curious why this re-formatting occurs. Does Biome have different settings per dev environment?